docs/access/alternatives.md

```diff
@@ -13,15 +13,15 @@ Using [`cvmfsexec`](https://github.com/cvmfs/cvmfsexec), mounting of CernVM-FS r
 an unprivileged user is possible, without having CernVM-FS installed system-wide.
 
 `cvmfsexec` supports multiple ways of doing this depending on the system configuration,
-where way requires that specific features are enabled in the system, like:
+depending on which specific features are enabled in the system, like:
```

Comment:
This is more or less saying the same thing twice, with two times `depending one`. Maybe the "depending on the system configuration," part can be removed?

Reply:
fixed in 642e053